### PR TITLE
More gr.render content

### PR DESCRIPTION
**Pull request by aliabid94**
Updated guide, two new complex demos:
todo list:
todof2

audio mixer:
audiomixer

### CODE REVIEW

File: guides/04_building-with-blocks/04_dynamic-apps-with-render-decorator.md
Comment (freddyaboulton) [inline guides/04_building-with-blocks/04_dynamic-apps-with-render-decorator.md:63]: typo: taht

**Review by freddyaboulton — APPROVED** [review]
🔥

**Review by aliabd — APPROVED** [review]
lgtm, the demos will upload after merging

**Pull request merged**
